sdk/go/common/workspace/templates.go

```diff
@@ -388,7 +388,7 @@ func RetrieveGitFolder(rawurl string, path string) (string, error) {
 		var cloneErr error
 		for _, ref := range refAttempts {
 			// Attempt the clone. If it succeeds, break
-			cloneErr = gitutil.GitCloneOrPull(url, ref, path, true /*shallow*/)
+			cloneErr := gitutil.GitCloneOrPull(url, ref, path, true /*shallow*/)
```

Comment:
This change disables error handling on L396 `if cloneErr != nil`, why?

Reply:
No reason - reverted this.